### PR TITLE
Remove unused arguments

### PR DESCRIPTION
**Pull request by Evangelink**

### CODE REVIEW

File: src/DataCollectors/Microsoft.TestPlatform.Extensions.EventLogCollector/EventLogDataCollector.cs [hunk 1]
```diff
@@ -383,7 +383,7 @@ private void RemoveTempEventLogDirs(List<string> tempDirs)
         }
     }
 
-    private void StartCollectionForContext(DataCollectionContext dataCollectionContext, bool isSessionContext)
```
Comment (nohwnd) [inline src/DataCollectors/Microsoft.TestPlatform.Extensions.EventLogCollector/EventLogDataCollector.cs:386]: Looks like there was some intent behind having this parameter, did someone from our current team write it? Can you confirm with them it is unused on purpose please?

Comment (nohwnd) [inline src/DataCollectors/Microsoft.TestPlatform.Extensions.EventLogCollector/EventLogDataCollector.cs:386]: Looks like there was some intent behind having this parameter, did someone from our current team write it? Can you confirm with them it is unused on purpose please?

File: src/Microsoft.TestPlatform.CrossPlatEngine/Execution/BaseRunTests.cs [hunk 2]
```diff
@@ -249,7 +249,7 @@ public void RunTests()
     internal void Abort()
     {
         EqtTrace.Verbose("BaseRunTests.Abort: Calling RaiseTestRunComplete");
-        RaiseTestRunComplete(exception: null, canceled: _isCancellationRequested, aborted: true, adapterHintToShutdownAfterRun: false, elapsedTime: TimeSpan.Zero);
```
Comment (nohwnd) [inline src/Microsoft.TestPlatform.CrossPlatEngine/Execution/BaseRunTests.cs:252]: This also looks like it has some purpose.

Comment (nohwnd) [inline src/Microsoft.TestPlatform.CrossPlatEngine/Execution/BaseRunTests.cs:252]: This also looks like it has some purpose.

File: src/DataCollectors/Microsoft.TestPlatform.Extensions.EventLogCollector/EventLogDataCollector.cs (hunk 1 above)
Comment (Evangelink) [reply src/DataCollectors/Microsoft.TestPlatform.Extensions.EventLogCollector/EventLogDataCollector.cs:386]: Method was added with the argument but it was never used so I cannot say if Harsh Jain was willing to do something with this back in 2017.

File: src/Microsoft.TestPlatform.CrossPlatEngine/Execution/BaseRunTests.cs [hunk 3]
```diff
@@ -580,7 +580,6 @@ private void SetAdapterLoggingSettings()
         Exception exception,
         bool canceled,
         bool aborted,
-        bool adapterHintToShutdownAfterRun,
```
Comment (Evangelink) [inline src/Microsoft.TestPlatform.CrossPlatEngine/Execution/BaseRunTests.cs:583]: Was also introduced without ever being used (2016).

File: test/TranslationLayer.UnitTests/VsTestConsoleRequestSenderTests.cs [hunk 4]
```diff
@@ -2735,7 +2735,7 @@ private void InitializeCommunication(int protocolVersion)
         Assert.IsTrue(connectionSuccess, "Connection must succeed.");
     }
 
-    private void SetupMockCommunicationForRunRequest(Mock<ITestRunEventsHandler> mockHandler)
+    private void SetupMockCommunicationForRunRequest()
```
Comment (Evangelink) [inline test/TranslationLayer.UnitTests/VsTestConsoleRequestSenderTests.cs:2738]: Was also introduced in 2017 without being used.

File: test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/Client/Parallel/ParallelProxyDiscoveryManagerTests.cs [hunk 5]
```diff
@@ -229,7 +229,7 @@ public void HandlePartialDiscoveryCompleteShouldCreateANewProxyDiscoveryManagerI
         Assert.IsTrue(_proxyManagerFuncCalled);
     }
 
-    private IParallelProxyDiscoveryManager SetupDiscoveryManager(Func<IProxyDiscoveryManager> getProxyManager, int parallelLevel, bool abortDiscovery, int totalTests = 20)
+    private IParallelProxyDiscoveryManager SetupDiscoveryManager(Func<IProxyDiscoveryManager> getProxyManager, int parallelLevel, bool abortDiscovery)
```
Comment (Evangelink) [inline test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/Client/Parallel/ParallelProxyDiscoveryManagerTests.cs:232]: Usage was remove in 2017 after a refactoring.

**Review by nohwnd — APPROVED** [review]

**Pull request merged**